pyth-sdk-solana/src/state.rs

```diff
@@ -1,4 +1,8 @@
 //! Structures and functions for interacting with Solana on-chain account data.
+//!
+//! The definitions here should be in sync with the definitinos provided within the pyth-oracle
+//! Solana contract. To enforce this the tests at the bottom of this file attempt to compare the
+//! definitions here with the definitions in the contract.
```

Comment:
why bother having the definitions here if you want them to be equivalent to the ones in pyth-oracle? seems like redundant code and we should just import from pyth-oracle.

Reply:
I'll fix the comment, they aren't identical, just isomorphic. I.E the byte layout is the same but the API exposed isn't the same. I didn't like that it wasn't easy to tell at a glance if those conversions were in sync.